core/kazoo_ast/src/kapps_config_usage.erl

```diff
@@ -194,26 +194,26 @@ key_to_key_path(?BINARY_MATCH(K)) ->
     end.
 
 guess_type('get_list', Default) -> guess_type_by_default(Default);
-guess_type('is_true', _Default) -> <<"boolean">>;
-guess_type('get_is_true', _Default) -> <<"boolean">>;
-guess_type('get_boolean', _Default) -> <<"boolean">>;
+guess_type('is_true', _) -> <<"boolean">>;
```

Comment:
Prefer to keep `_Default` as it helps the reader remember what is in the arg position.